Cargo.toml

```diff
@@ -16,6 +16,7 @@ name = "mycelium_kernel"
 
 [dependencies]
 hal-core = { path = "hal-core" }
+log = { version = "0.4", default_features = false }
```

Comment:
It might be worth re-exporting the `log` macros from `mycelium_util`, so we don't need to remember to set `default_features = false` on every crate where we want to use logging macros. Not sure though.

Reply:
not a bad idea!

Reply:
hmm the downside to this is that the macros are now imported from `mycelium_util`, which feels a little weird...i like how with `log::info!`/`tracing::info!`, the crate name describes what the macro does...